src/chart/MultiChart.js

```diff
@@ -51,16 +52,22 @@
         { id: "GOOGLE_COLUMN", display: "Column (Google)", widgetClass: "google_Column" },
         { id: "GOOGLE_LINE", display: "Line (Google)", widgetClass: "google_Line" },
         { id: "GOOGLE_SCATTER", display: "Scatter (Google)", widgetClass: "google_Scatter" },
+        { id: "GOOGLE_GANTT", display: "Gantt (Google)", widgetClass: "google_Gantt" },
```

Comment:
its going to by the time 1.10 is released ... just planning ahead ... ill have it done end of week